### DISCUSSION TITLE
be able  to change main target in cayo

### DISCUSSION THREAD

**Discussion by snwyyy** [YimMenu/YimMenu#1488] [Ideas]
it would be pretty chill to be able to change it

**Comment by SlavicBlin** [comment]
counts as money method i belive

**Reply by snwyyy** [reply]
ah okay

**Comment by SlavicBlin** [comment]
someone should just make a recovery version of the menu :D

**Reply by Tayipc38** [reply]
Could be done with the lua's in a personal or community capacity.

**Reply by SlavicBlin** [reply]
yeah if you know what you are doing cuz me personally i have no idea how lua works

**Reply by snwyyy** [reply]
idk how that works either

**Reply by snwyyy** [reply]
how do i do it with lua?
